File: problemSets/pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float i;
    do
    {
        i = get_float("Change owed: ");
    }
    while (i <= 0);
    int j = round(i * 100);
    int d = (j / 25);
    int c = (j % 25);
    int b = (c % 10);
    int a = (b % 5);
    int f = (d + (c / 10) + (b / 5) + a);
    printf("%i\n", f);
}

File: problemSets/pset6/cash/cash.py
from cs50 import get_float


i = 0.0
while i < 0.01:
    i = get_float("Change owed: ")

j = int(round(i * 100))
d = int(j / 25)
c = int(j % 25)
b = int(c % 10)
a = int(b % 5)
f = int(d + (c / 10) + (b / 5) + a)
print(f)

